docs/reference/copy-guide.md

```diff
@@ -0,0 +1,275 @@
+“Click “settings” to access the advanced settings.”
+
+We sometimes think that adding the extra words helps the user feel like they’re being walked through the product, but most of the time, they really are just extra words. 
+
+Similar are “Once you do X…” or “If you want to do X.” Sometimes there are decision points where “If you want to do X” is entirely appropriate and needed because there are different paths the user can take based on their goal, but we often use it to mean “Here is a thing you can do,” which you can express more simply with something like “To do X…”
+
+#### THREE: Be specific.
+When an action depends on the user having completed some prior action, be specific about what’s required and what happens next. We often default to “when you’re ready.”
+Ready for what? Be specific about whatever the prerequisites are.
+
+“When you’re ready” can mean:
+
+“When you want to add another block”
+“When you’re satisfied with your post”
+“After you’ve finished proofreading your post”
+“When you’d like to add a featured image”
+“After you’ve configured all the settings”
+
+And when something means everything, it actually means nothing. The more specific instructions are, the more useful they are, and the more trust the person following them will have in the product.
+
+#### FOUR: This is still writing. It should have personality and interest.
+Clarity above all, yes, and space is often limited here -- but UI text can still be interesting to read.
+
+Single lines of description can still be complete sentences. 
+
+List. Numbered or bulleted.
+
+vs.
+
+Add a list, either numbered or bulleted.
+
+You can still use contractions. 
+
+Add a list. We will provide formatting options.
+
+vs.
+
+Add a bulleted list -- we’ll give you some formatting options.
+
+You can still use punctuation -- em dashes, colons, semicolons -- to control the flow of your words, link ideas, and create pauses. 
+
+List. Numbered or bulleted.
+
+vs.
+
+Add a list -- numbered or bulleted. Your choice!
+
+You can still try to avoid jargon in favor of plain language.
+
+Add unordered or ordered list.
+
+vs.
+
+Add a list, either numbered or bulleted.
+
+(And because it bears repeating: no wordplay, please! “Personality” can -- and in UI instructions, should -- be subtle. We’re talking about text that sounds like it was said by a human being, not forced attempts at whimsy.)
+
+#### FIVE: Pay attention to capitalization.
+When it comes to headlines and subheads, there are two ways to capitalize:
+
+In Title Case, the First Letter of Almost Every Word Is Capitalized
+
+In sentence case, only the first letter of the line is capitalized
+
+Feature names and dashboard sections typically use title case (think “Site Stats” or “Recently Published”), whereas feature labels typically use sentence case (like “Show buttons on” or “Comment Likes are,” where “Likes” is capitalized because it’s the feature name, but the overall label is using sentence case).
```

Comment:
Could we clarify here whether or not we should use sentence case or title case for UI controls such as buttons? I'm often confused about which style to use when developing features. There's sometimes conflicting examples throughout the interface.

<img width="281" alt="screen shot 2018-06-28 at 10 06 37" src="https://user-images.githubusercontent.com/612155/42006104-28b9c63c-7abb-11e8-9e80-33f456ce49b4.png">
<img width="299" alt="screen shot 2018-06-28 at 10 08 16" src="https://user-images.githubusercontent.com/612155/42006114-33632d1c-7abb-11e8-97a1-109b1a2acc55.png">
<img width="264" alt="screen shot 2018-06-28 at 10 06 46" src="https://user-images.githubusercontent.com/612155/42006106-29cba2a2-7abb-11e8-94f1-6a8a75611740.png">
<img width="244" alt="screen shot 2018-06-28 at 10 06 53" src="https://user-images.githubusercontent.com/612155/42006108-29f6dbac-7abb-11e8-9ea3-a407bf286f5f.png">

Reply:
It's actually not even clear what case these are in! They could be:
1. Title case
2. Sentence case, but with capitalized the block names, so they effectively look like title case

We should first update to specific whether block names are/are not proper nouns (and therefore always capitalized), and then y'all should decide what case you prefer for these controls.

Fun with capitalization :)